QUARKUS-960.md

```diff
@@ -0,0 +1,29 @@
+# Quarkus-960 - Improved getting started experience with CLI
+JIRA link: https://issues.redhat.com/browse/QUARKUS-960
+
+There was an existing CLI tool in Quarkus that has been fully reworked as part of QUARKUS-960. 
+
+So, the goal is to verify proper functionality of the new Quarkus CLI tool.
+
+## Scope of the testing
```

Comment:
@rsvoboda what about the CI delivery mechanism and documentation? These points would not be covered by test automation, but manual verification, so I'm not sure if those need to be noted here.

Reply:
Maybe add dedicated section for manual checks near the end